dipy/align/parzenhist.pyx

```diff
@@ -619,10 +619,10 @@ cdef _compute_pdfs_dense_2d(double[:, :] static, double[:, :] moving,
         cnp.npy_intp offset, valid_points
         cnp.npy_intp i, j, r, c
         double rn, cn
-        double val, spline_arg, sum
+        double val, spline_arg, s
```

Comment:
We try to avoid 1 letter variable (apart from maths equation). Can you rename it? `total, summation, ...`

Reply:
Done 4e9cab5.